Readme.md

```diff
@@ -10,7 +10,7 @@ Locally can run on regtest.
 
 ## Running locally
 
-Supposing to have [rust installed](https://www.rust-lang.org/tools/install) >= 1.65.0 and a synced [bitcoin core](https://bitcoincore.org/en/download/) on mainnet with `txindex=1` and `rest=1` do:
+Supposing to have [rust installed](https://www.rust-lang.org/tools/install) > 1.65.0 and a synced [bitcoin core](https://bitcoincore.org/en/download/) on mainnet with `txindex=1` and `rest=1` do:
```

Comment:
could you please use "1.66.0": (either "at least 1.66.0" or ">= 1.66.0") I think it's more clear

Reply:
Done.